### DISCUSSION TITLE
System wide search

### DISCUSSION THREAD

**Discussion by bluebrad** [Leantime/leantime#1625] [Ideas]
I would like to see a system wide search option. so i can search documents and all posts. and have filters for only current projects in selection or search all. for example i uploaded a graphic with a unique name but i do not know what the project was for. i know its in the documents of one of the projects but with over 80 projects its going to take me some time to look for the graphic. if there was a search box i can search for the graphics name and find the project and notes left with it.

**Comment by marcelfolaron** [comment]
Thank you for the request. A system wide search is on our radar and will be implemented. I can't commit to a timeline just yet but will update as soon as I have a clear path
